### PR TITLE
Use upstream auth0/samlp v4.0.0 instead of fork

### PR DESCRIPTION
**Pull request by crolarlibertyva**
With this PR we can conclude API-3112 with referencing the original upstream project.

### CODE REVIEW

File: saml-proxy/package-lock.json [hunk 1]
```diff
@@ -7180,9 +7180,9 @@
       }
     },
     "moment": {
-      "version": "2.27.0",
-      "resolved": "https://registry.npmjs.org/moment/-/moment-2.27.0.tgz",
-      "integrity": "sha512-al0MUK7cpIcglMv3YF13qSgdAIqxHTO7brRtaz3DlSULbqfazqkc5kEjNrLDOM7fsjshoFIihnU8snrP7zUvhQ=="
+      "version": "2.19.3",
```
Comment (dbrown-va) [inline saml-proxy/package-lock.json:7183]: I vaguely remember a dependabot upgrade of this dependency over the summer.

Does this return any issues with an `npm audit`?

Comment (crolarlibertyva) [reply saml-proxy/package-lock.json:7183]: Not for this one, at least not at the critical level. But will look again, just to make sure it wasn't lost to the noise from the node-forge flag.

Comment (crolarlibertyva) [reply saml-proxy/package-lock.json:7183]: Confirmed, nothing on moment.

Comment (crolarlibertyva) [reply saml-proxy/package-lock.json]: Went ahead and used resolutions to ensure the upgrade to above what it was prior.

Comment (crolarlibertyva) [reply saml-proxy/package-lock.json:7183]: Undid that resolution which causes version to turn into 2.19.3.

**Review by bdgrider — APPROVED** [review]
All appears to be working on my checkout of this branch.

**Comment by beaugrantham-va** [comment]
Closed in favor of https://github.com/department-of-veterans-affairs/vets-saml-proxy/pull/166

**Pull request closed without merging**